app/mailers/multi_notifications.rb

```diff
@@ -0,0 +1,15 @@
+class MultiNotifications < ApplicationMailer
+  def self.consultation_deadline_upcoming(consultation, weeks_left:, mailer: Notifications)
+    addresses = consultation.authors.uniq.map(&:email)
```

Comment:
You could do a `consultation.authors.pluck(:email).uniq` here which would be more efficient